16-projects.Rmd

```diff
@@ -265,6 +265,45 @@ Note that you need to change the vignette title in both the `title` field and th
 
 The vignette output format does not have to be HTML. It can also be PDF, so you can use `output: pdf_document`, too. Any other output formats that create HTML or PDF are also okay, such as `beamer_presentation` and `tufte::tufte_html`. However, currently R only recognizes HTML and PDF vignettes.
 
+## R Markdown templates
+
+Figure \@ref(fig:package-vignette) of section \@ref(package-vignette) illustrates the process of retrieving the editable "Package Vignette (HTML)" template from the **rmarkdown** package. This R Markdown file is pre-populated with the appropriate metadata for an R package vignette. 
+
+Similarly, any package may include R Markdown templates whcih package users can access through the RStudio IDE (as shown in the figure) or across any platform with the `rmarkdown::draft()` function.
+
+### Template use-cases
+
+Templates are a useful way to share custom structure, style, and content. There are many excellent examples of this "in the wild".
+
+Many templates add structure and style by pre-populating the YAML metadata. We already saw an example of this with the **rmarkdown** package's "Package Vignette (HTML)" template. Similarly, the **rmdformats** package provides a number of templates which pass different custom styling functions to the `output` option. 
+
+Other templates demonstrate required document structure. For example, the **pagedown** package includes numerous templates for posters, resumes, and other page layouts Similarly, the **xaringan** package's "Ninja Presentation" template demonstrates the syntax for many different slide formatting options. 
+
+Templates may also demonstrate package features and syntax. For example, the **flexdashboard** package and the **learnr** package both include templates with code chunks that call functions from the packages to create a sample dashboard or tutorial, respectively. 
+
+Similarly, templates may also include boilerplate content. For example, the **rticles** package provides many such templates to align R Markdown output to the required style and content guidelines of different academic journals. Boilerplate content is also useful in in organizational settings, such as a team generating a quarterly reports. One example of,    
```

Comment:
> One example of,    

This sentence is incomplete.

Reply:
Thanks for catching and sorry for the sloppiness there! I rearranged that paragraph to be more readable, but accidentally missed that part in clean up. Deleted now.

Reply:
No worries! This does happen frequently in writing. This would actually be the job of our copy editor, but since I'm too familiar with her requirements, I'm just doing quite a bit of her job, and hoping she wouldn't find more than 20 issues in my fifth book with her 😉